starters/next-chakra-ui/src/pages/index.tsx

```diff
@@ -42,7 +42,7 @@ const Home: NextPage = () => {
             p={4}
             bg="#2464ec"
           >
-            NextJS 12 and Chakra UI Starter Kit
+            NextJS and Chakra UI Starter Kit
```

Comment:
The text for `index.test.tsx` needs to be updated too because the test is currently failing.